src/pages/Models/Models.js

```diff
@@ -22,19 +22,21 @@ export default function Models() {
   const queryStrings = queryString.parse(location.search);
   // If it doesn't exist, fall back to grouping by status
   const groupedByFilter = queryStrings.groupedby || "status";
-  // Set initial state using filter from URL
-  const [groupedBy, setGroupedBy] = useState(groupedByFilter);
-  // Assign updated filter back to queryStrings obj
-  queryStrings.groupedby = groupedBy;
-  // Stringify the obj again to push back to URL
-  const updatedQs = queryString.stringify(queryStrings);
-  // Add as an effect so UI is updated on each component render (e.g. Back button)
-  useEffect(() => {
+
+  const [groupModelsBy, setGroupModelsBy] = useState(groupedByFilter);
```

Comment:
@barrymcgee FYI Even though `groupedByFilter` was changing with the back button interactions, because the initial state of the state was already set it didn't update the `groupModelsBy` state. So the following 3 lines are used to do just that.